helm/cert-manager-app/values.yaml

```
@@ -9,7 +9,7 @@ global:
 
 image:
   name: giantswarm/cert-manager-controller
-  tag: v0.9.0
+  tag: v0.15.1
```

```suggestion
  # when updating the tag please make sure to keep appVersion in Chart.yaml in sync
  tag: v0.15.1
```

@glitchcrab please update appVersion in Chart.yaml too

---

will do, this will be quite a big PR with all necessary changes so i'm just working through in small increments :+1:

---

There are many upgrade instructions we need to compress into single release notes and single migration process:
- https://cert-manager.io/docs/installation/upgrading/upgrading-0.9-0.10/
- https://cert-manager.io/docs/installation/upgrading/upgrading-0.10-0.11/
- https://cert-manager.io/docs/installation/upgrading/upgrading-0.11-0.12/
- https://cert-manager.io/docs/installation/upgrading/upgrading-0.12-0.13/
- https://cert-manager.io/docs/installation/upgrading/upgrading-0.13-0.14/
- https://cert-manager.io/docs/installation/upgrading/upgrading-0.14-0.15/

---

yeah, i'm working my way through those - there are going to be a fair few manual interventions required